vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -0,0 +1,104 @@
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-write-vnodes', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['test_write_gradual_increase_load'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-during-rolling-upgrade', region: 'eu-west-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['test_mixed_gradual_increase_load'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-with-nemesis', region: 'eu-west-2', versions: ['2024.1', '2024.2', '2025.1', '2025.2'], sub_tests: ['test_mixed_gradual_increase_load'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-with-nemesis', region: 'eu-west-2', versions: ['master'], sub_tests: ['test_mixed_gradual_increase_load', 'test_read_gradual_increase_load', 'test_write_gradual_increase_load'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-with-nemesis-rbno-disabled', region: 'eu-west-3', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['test_mixed_gradual_increase_load'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_arm64', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_arm64-write', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_x86_64', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_x86_64-write', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-master/perf-regression/perf-regression-predefined-throughput-steps-sanity-vnodes', region: '', versions: [], sub_tests: [], scheduler: ''],
```

Comment:
should this gonna have `schedule`  like `master-build` or `master-daily`

Reply:
done